### PR TITLE
feat(settings): Create content-server version of AppLayout and move styles

### PR DESCRIPTION
**Pull request by LZoog**
This commit:
* Adds AppLayout with Tailwind styles and moves Moz logo from fxa-settings into fxa-react for use in Reactified content-server routes
* Adds postcss-assets to settings for future background-images
* Moves .card styles into [fxa-react](https://mozilla-hub.atlassian.net/browse/FXA-react)
* README update on inlining SVGs vs src

---

Closes FXA-6275

~*Issue I'll be filing*:  we should be able to reference `url('whatever')` in our background-images in Tailwind but currently we inline everything. I think inlining in most cases makes sense, but sometimes it makes sense to just let the network request happen so images can be cached, e.g. the `ff-logo` makes sense to cache because we use it everywhere and it's a heavy SVG. We can only let this caching happen on optional use of `img` right now.~ Never mind, apparently we're already set up to do this. I updated the README to reflect this.

Note: if the `min-h-screen` forces a scrollbar to show outside of Storybook I'll tweak that when I see it.

You'll only be able to view in Storybook for now. Keep in mind Storybook adds additional padding for stories, here's screenshots without:
image
image

### CODE REVIEW

**Comment by pdehaan** [comment]
Q: Should `packages/fxa-react/images/mozilla.svg` have transparent text, or should we make the text white?
Probably moot at this point, but might be interesting when/if we support dark browser themes.

**Comment by LZoog** [comment]
> Q: Should packages/fxa-react/images/mozilla.svg have transparent text, or should we make the text white?
Probably moot at this point, but might be interesting when/if we support dark browser themes.

Haha good thinking. ;) I thought about that, and I'm pretty sure we'll have to use a different image to flip the black to white... but we might be able to change the path colors with CSS. I'll see what payments' `mozilla.svg` looks like, there's one in that package (that can probably reference the one in `fxa-react` I'm adding here)

File: packages/fxa-react/components/LinkExternal/index.tsx [hunk 1]
```diff
@@ -18,15 +19,16 @@ export const LinkExternal = ({
   children,
   title,
   'data-testid': testid = 'link-external',
+  rel = 'noopener noreferrer',
```
Comment (millsoper) [inline packages/fxa-react/components/LinkExternal/index.tsx:22]: nice

**Review by millsoper — APPROVED** [review]
Looks perfect! Thank you @LZoog !

**Pull request merged**